Tasks/AzureResourceGroupDeploymentV2/README.md

```diff
@@ -73,6 +73,28 @@ The following parameters are shown when the selected action is to create or upda
  
  * **Deployment Mode**: This specifies the [deployment mode](https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-deploy) in which the Azure resources specified in the template have to be deployed. Incremental mode handles deployments as incremental updates to the resource group . It leaves unchanged resources that exist in the resource group but are not specified in the template. Complete mode deletes resources that are not in your template. [Validate mode](https://msdn.microsoft.com/en-us/library/azure/dn790547.aspx) enables you to find syntactical problems with the template before creating actual resources. By default, incremental mode is used. 
  
+ ### Deployment Outputs:
+  It is the output of the deployment object created using Azure Resource Manager templates which can be used in the subsequent tasks like Powershell and Azure CLI.
```

Comment:
Re-word to "[Outputs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-authoring-templates#outputs) created by Azure Resource Manager template deployment. It can be used in subsequent tasks (like Azure Powershell or Azure CLI) for further processing."

Reply:
Done